### ISSUE TITLE
How to get the estimated Adjacency matrix

### ISSUE THREAD

**Issue by SaqibMamoon** [ban-epfl/rcd#2]
Hi, 

Is it possible to get the adjacency matrix as the output instead of the list?

**Comment by sepehr78** [comment]
What do you mean by the list? The `learn_and_get_skeleton` function returns a NetworkX undirected graph.

**Comment by SaqibMamoon** [comment]
@sepehr78 

Yes, I got your point; thanks a lot for the clarification.  I converted the graph object to numpy Adj matrix using nx.to_numpy_array. However, it returns the binary matrix, is it possible to get the weighted matrix?
